pom.xml

```diff
@@ -55,15 +55,15 @@
         <kotlin.version>1.6.10</kotlin.version>
         <kotlin.coroutines.version>1.6.0</kotlin.coroutines.version>
         <neo4j.version>4.4.3</neo4j.version>
-        <kafka.version>2.4.1</kafka.version>
+        <kafka.version>3.5.1</kafka.version>
```

Comment:
Let's stick with 2.6.3 as we discussed before with @conker84.

Reply:
done